ingest/common/domain/errors.go

```diff
@@ -1,4 +1,4 @@
-package domain
+package commondomain
```

Comment:
Can we keep `domain` package and let client to create alias import name, IMHO it's super confusing when last part of the import path differs from defined package, and it's more clear to let cilent to define import alias if needed:

```
import "https://github.com/osmosis-labs/osmosis/ingest/common/domain"

// ..

commondomain.Do()
```

vs 

```
import commondomain "https://github.com/osmosis-labs/osmosis/ingest/common/domain"

// ...

commondomain.Do
```

Reply:
That makes sense. Will update it. 👍

Reply:
Done